Bug in census checks when using the HTTP API

When running the single lar validation API, the census checks fail due to a `FileNotFoundException`. There seems to be a problem with the relative path of the `tract_to_cbsa_2013.csv` file

```shell
[ERROR] Caused by: java.io.FileNotFoundException: model/src/main/resources/tract_to_cbsa_2013.csv (No such file or directory)
api[ERROR] 	at java.io.FileInputStream.open0(Native Method)
api[ERROR] 	at java.io.FileInputStream.open(FileInputStream.java:195)
api[ERROR] 	at java.io.FileInputStream.<init>(FileInputStream.java:138)
api[ERROR] 	at scala.io.Source$.fromFile(Source.scala:91)
api[ERROR] 	at scala.io.Source$.fromFile(Source.scala:76)
api[ERROR] 	at hmda.model.census.CBSATractLookup$.<init>(CBSATractLookup.scala:9)
api[ERROR] 	at hmda.model.census.CBSATractLookup$.<clinit>(CBSATractLookup.scala)
```

--- a/model/src/main/scala/hmda/model/ResourceUtils.scala
+++ b/model/src/main/scala/hmda/model/ResourceUtils.scala
@@ -1,0 +1,16 @@
+package hmda.model
+
+import scala.io.{ BufferedSource, Source }
+
+trait ResourceUtils {
+  // For opening and reading files in this project's /resources directory.
+
+  def resource(filename: String): BufferedSource = {
+    val file = getClass.getResourceAsStream(filename)
+    Source.fromInputStream(file)
+  }
+
+  def resourceLines(filename: String): Iterator[String] = {
+    resource(filename).getLines()
+  }
+}

--- a/model/src/main/scala/hmda/model/census/CBSAMetroMicroLookup.scala
+++ b/model/src/main/scala/hmda/model/census/CBSAMetroMicroLookup.scala
@@ -1,13 +1,10 @@
 package hmda.model.census
 
-import java.io.File
+import hmda.model.ResourceUtils
 
-import scala.io.Source
-
-object CBSAMetroMicroLookup {
+object CBSAMetroMicroLookup extends ResourceUtils {
   val values: Seq[CBSAMetroMicro] = {
-    val cbsaResource = getClass.getResourceAsStream("/tl_2013_us_cbsa.csv")
-    val lines = Source.fromInputStream(cbsaResource).getLines()
+    val lines = resourceLines("/tl_2013_us_cbsa.csv")
 
     lines.map { line =>
       val values = line.split('|').map(_.trim)

--- a/model/src/main/scala/hmda/model/census/CBSAMetroMicroLookup.scala
+++ b/model/src/main/scala/hmda/model/census/CBSAMetroMicroLookup.scala
@@ -6,7 +6,9 @@ import scala.io.Source
 
 object CBSAMetroMicroLookup {
   val values: Seq[CBSAMetroMicro] = {
-    val lines = Source.fromFile(new File("model/src/main/resources/tl_2013_us_cbsa.csv")).getLines
+    val cbsaResource = getClass.getResourceAsStream("/tl_2013_us_cbsa.csv")
+    val lines = Source.fromInputStream(cbsaResource).getLines()
+
     lines.map { line =>
       val values = line.split('|').map(_.trim)
       val combinedCode = values(0)

--- a/model/src/main/scala/hmda/model/census/CBSATractLookup.scala
+++ b/model/src/main/scala/hmda/model/census/CBSATractLookup.scala
@@ -6,7 +6,9 @@ import scala.io.Source
 
 object CBSATractLookup {
   val values: Seq[CBSATract] = {
-    val lines = Source.fromFile(new File("model/src/main/resources/tract_to_cbsa_2013.csv")).getLines
+    val cbsaResource = getClass.getResourceAsStream("/tract_to_cbsa_2013.csv")
+    val lines = Source.fromInputStream(cbsaResource).getLines()
+
     lines.map { line =>
       val values = line.split('|').map(_.trim)
       val name = values(0)

--- a/model/src/main/scala/hmda/model/census/CBSATractLookup.scala
+++ b/model/src/main/scala/hmda/model/census/CBSATractLookup.scala
@@ -1,13 +1,10 @@
 package hmda.model.census
 
-import java.io.File
+import hmda.model.ResourceUtils
 
-import scala.io.Source
-
-object CBSATractLookup {
+object CBSATractLookup extends ResourceUtils {
   val values: Seq[CBSATract] = {
-    val cbsaResource = getClass.getResourceAsStream("/tract_to_cbsa_2013.csv")
-    val lines = Source.fromInputStream(cbsaResource).getLines()
+    val lines = resourceLines("/tract_to_cbsa_2013.csv")
 
     lines.map { line =>
       val values = line.split('|').map(_.trim)

